metadata.json

```diff
@@ -99,6 +99,14 @@
     {
       "name": "herculesteam-augeasproviders",
       "version_requirement": ">= 0.5.1 < 3.0.0"
+    },
+    {
+      "name": "herculesteam-augeasproviders_core",
```

Comment:
These dependencies aren't needed in the metadata.json as they are already dependencies of `herculesteam-augeasproviders`.

From https://github.com/hercules-team/augeasproviders/blob/d04c6df5f93870ded65084a623d12b4dc86db4d3/README.md
> This module provides is a meta module which gathers all official augeasproviders modules as dependencies.